Sources/MessagingInApp/Gist/Managers/MessageManager.swift

```diff
@@ -25,6 +26,9 @@ class MessageManager: EngineWebDelegate {
         self.currentMessage = message
         self.currentRoute = message.messageId
 
+        let diGraph = DIGraphShared.shared
+        self.logger = diGraph.logger
```

Comment:
again, is there any benifit of using `diGraph.logger` here rather than just initializing the logger with `DIGraphShared.shared`

Reply:
Valid point. However, this is only done in classes that are moved to `DIGraphShared` in PR #802. Using `DIGraphShared` in constructor slightly simplified the changes in that PR. Does this make sense?